apps/studio/components/interfaces/Storage/VectorBuckets/VectorBucketDetails.tsx

```diff
@@ -260,39 +264,63 @@ export const VectorBucketDetails = () => {
                                       type="default"
-                                      className="px-1"
+                                      className="w-7"
                                       icon={<MoreVertical />}
                                       onClick={(e) => e.stopPropagation()}
                                     />
                                   </DropdownMenuTrigger>
                                   <DropdownMenuContent side="bottom" align="end" className="w-40">
+                                    {wrapperInstance ? (
+                                      <>
+                                        <DropdownMenuItem
+                                          className="flex items-center space-x-2"
+                                          asChild
+                                        >
+                                          {/* TODO: Proper URL for sql editor */}
+                                          <Link
+                                            href={`/project/${projectRef}/sql/new?content=${encodeURIComponent(`select * from "${getVectorBucketFDWSchemaName(bucketId!)}"."${name}";`)}`}
+                                            onClick={(e) => e.stopPropagation()}
+                                          >
+                                            <SqlEditor
+                                              size={12}
+                                              className="text-foreground-lighter"
+                                            />
+                                            <p>Query in SQL Editor</p>
+                                          </Link>
+                                        </DropdownMenuItem>
+                                        <DropdownMenuItem
+                                          className="flex items-center space-x-2"
+                                          asChild
+                                        >
+                                          {/* TODO: Proper URL for table editor */}
+                                          <Link
+                                            href={`/project/${projectRef}/editor/${encodeURIComponent(
+                                              name
+                                            )}?schema=${getVectorBucketFDWSchemaName(bucketId!)}`}
+                                            onClick={(e) => e.stopPropagation()}
+                                          >
+                                            <TableEditor
+                                              size={12}
+                                              className="text-foreground-lighter"
+                                            />
+                                            <p>View in Table Editor</p>
+                                          </Link>
+                                        </DropdownMenuItem>
+                                      </>
+                                    ) : null}
```

Comment:
Ideally both of these:

- Query in SQL Editor
- View in Table Editor

...get moved to sibling foreign row beneath this one, after we match the analytics buckets UI merged in https://github.com/supabase/supabase/pull/40645